src/release_notes_generator.py

```diff
@@ -0,0 +1,196 @@
+import json
+import logging
+import traceback
+
+from typing import Optional
+from github import Github, Auth
+
+from github_integration.gh_action import get_action_input, set_action_output, set_action_failed
+from github_integration.github_manager import GithubManager
+
+from release_notes.formatter.record_formatter import RecordFormatter
+from release_notes.model.custom_chapters import CustomChapters
+from release_notes.model.record import Record
+from release_notes.release_notes_builder import ReleaseNotesBuilder
+from release_notes.factory.record_factory import RecordFactory
+
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+github_manager = GithubManager()
```

Comment:
why this one here? It's global, I'd be careful about global variables or objects - they get executed whenever you would import this module (I know that it's not the case here but still I like to avoid such global objects)

Reply:
This line is wrong. I removed it the first I saw it.